### PR TITLE
DB parameters in AppProperties

### PR DESCRIPTION
**Pull request by l4v7l**

### CODE REVIEW

File: backend/src/main/java/com/gridstudentpractice/chatservice/AppProperties.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    @Value(("${database.password}"))
+    private String password;
+
+    public String getDatabaseUrl() {
+        System.out.println(url);
+        return url;
+    }
+
+    public String getDatabasePassword() {
+        System.out.println(password);
```
Comment (Mantarus) [inline backend/src/main/java/com/gridstudentpractice/chatservice/AppProperties.java]: Oh oh that's not secure at all!
Do you need these printlns?

File: backend/src/main/java/com/gridstudentpractice/chatservice/AppProperties.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties
+public class AppProperties {
+
+    @Value(("${database.url}"))
+    private String url;
```
Comment (Mantarus) [inline backend/src/main/java/com/gridstudentpractice/chatservice/AppProperties.java]: Let's keep namings consistent
url -> databaseUrl
Same for other fields

File: backend/src/main/java/com/gridstudentpractice/chatservice/AppProperties.java (hunk 1 above)
Comment (l4v7l) [reply backend/src/main/java/com/gridstudentpractice/chatservice/AppProperties.java]: no, it was just for test

File: backend/src/main/java/com/gridstudentpractice/chatservice/AppProperties.java (hunk 2 above)
Comment (ktrof) [reply backend/src/main/java/com/gridstudentpractice/chatservice/AppProperties.java]: done

**Pull request merged**